assets.md

```diff
@@ -165,6 +166,21 @@ but these individual OpenTelemetry members have been given rights to publish und
 Link: https://pypi.org/user/opentelemetry/
 
 - Owner: [@codeboten](https://github.com/codeboten)
+ 
+### PHP Extras
+
+##### SIG Extra Repositories: https://github.com/opentelemetry-php
+
+These read only repository clones are used in order to create split-repos, which are necessary for packagist php package publishing.
```

Comment:
can you add owners to this section also (even if the same)?

Reply:
@bobstrecansky please take a look